packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -17,12 +17,14 @@ interface SwitchMetadata {
   readonly missingBranchTypes: ts.Type[];
   readonly defaultCase: TSESTree.SwitchCase | undefined;
   readonly isUnion: boolean;
+  readonly containsNonLiteralType: boolean;
 }
 
 type Options = [
   {
     /**
-     * If `true`, allow `default` cases on switch statements with exhaustive cases.
+     * If `true`, allow `default` cases on switch statements with exhaustive
+     * cases.
```

Comment:
(this is fine I guess, no strong preference either way)

Reply:
the comment was "illegal" such that it passed the length of the ruler

Reply:
if you are interested in linting (and autofixing!) such comments, i wrote this: https://github.com/IsaacScript/isaacscript/blob/main/packages/eslint-plugin-isaacscript/docs/rules/format-jsdoc-comments.md

Reply:
Nifty, might be good as a rule in eslint-plugin-jsdoc?

If you want to file an issue suggesting using it here I'd be interested in that too 😄. Need to think on it a bit (are there edge cases?) but in general I like the idea!